descartes_moveit/src/seed_search.cpp

```diff
@@ -0,0 +1,455 @@
+﻿/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2015, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * This file defines methods that can be used to generate seed values for iterative
```

Comment:
Is there a better place for the description.  Perhaps in the header it would be picked up the doxygen generator.